plugin/client_rpc.go

```diff
@@ -610,25 +610,25 @@ func (s *hooksRPCServer) MessageWillBeUpdated(args *Z_MessageWillBeUpdatedArgs,
 	return nil
 }
 
-type Z_LogDebugArgs struct {
+type ZLogDebugArgs struct {
```

Comment:
@crspeller - Will there be any issues with changing these exported structs? Are there consumers depending on these?

Reply:
I think it should be fine. I think these are only exported for serialization (the reflect limitation). 
But I could be wrong. Someone should test to make sure plugins work on this branch. Break should be super obvious. 

Also a reminder that while the changes to this file will stick. The changes in  `plugin/client_rpc_generated.go` will not as that file is generated by `interface_generator`. That generator will have to be modified.

Reply:
Oh wow, that's a great catch, @haardikdharma10 did you manually modify each and every line of `plugin/client_rpc_generated.go`? I totally missed the fact that `plugin/interface_generator/main.go` is not touched at all :facepalm: 

Let's change that and then generate the file.

Reply:
Yes @agnivade! It was as simple as replacing all `Z_` with `Z`. However, I see that changing [this line](https://github.com/mattermost/mattermost-server/blob/master/plugin/interface_generator/main.go#L473) in `plugin/interface_generator/main.go` is all I have to do here. I have replaced it from `Z_ + name` to `Z + name`. How do i generate the `plugin/client_rpc_generated.go` file now?

Reply:
@haardikdharma10 `make pluginapi` should do it.

Reply:
@crspeller can't figure out what's with the failing check? 🤔